static/src/javascripts/projects/common/modules/experiments/tests/sign-in-gate-patientia.js

```diff
@@ -1,5 +1,5 @@
 // @flow
```

Comment:
it's not letting me leave a comment on the line,

the audience offset should be changed so that it doest overlap with the 100% test, so `audienceOffset` should be 0.9999

Reply:
so both at audience and offset 0.9999?